rllib/utils/replay_buffers/utils.py

```diff
@@ -52,11 +53,34 @@ def update_priorities_in_replay_buffer(
             #  policies (note: fixing this in torch_policy.py will
             #  break e.g. DDPPO!).
             td_error = info.get("td_error", info[LEARNER_STATS_KEY].get("td_error"))
+
+            policy_batch = train_batch.policy_batches[policy_id]
             # Set the get_interceptor to None in order to be able to access the numpy
             # arrays directly (instead of e.g. a torch array).
-            train_batch.policy_batches[policy_id].set_get_interceptor(None)
+            policy_batch.set_get_interceptor(None)
             # Get the replay buffer row indices that make up the `train_batch`.
-            batch_indices = train_batch.policy_batches[policy_id].get("batch_indexes")
+            batch_indices = policy_batch.get("batch_indexes")
```

Comment:
Is this change of code related to this PR?

Reply:
This is from #27544 .
This PR surfaced a bug in RNNsac that we did not catch before.
I've separated the PRs but included #27544 in this PR for the tests to pass already.

Reply:
I see. Do we have a unittest that would catch the old bug now?